src/components/pages/Navigation.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom';
+import NameForm from './StatedNameForm';
+import GraphForm from './StatedGraphForm';
 
 const Navigation = () => {
   return (
     <div>
-      <NavLink to="/">Home </NavLink>
-      <NavLink to="/graph">graph</NavLink>
+      <h2>
+        <BrowserRouter>
+          <h2>Welcome to cashburn, app for budget management</h2>
+          <li>
+            <Link to="/" className="nav-link">
+              Home
+            </Link>
+          </li>
+          <li>
+            <Link to="/graph" className="nav-link">
+              Graph
+            </Link>
+          </li>
+          <Switch>
+            <Route path="/" component={NameForm} exact />
```

Comment:
What do you think about use "exact" keyword? in Route path "/" ? 
beacuse path '/' include path '/' graph